content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+Taking another step to embrace interaction with a setup where participants had to repeatedly build Lego cars in pairs (though without transmission), @mcgraw_cultures_2014 take the complete joint interaction to be their object of study and propose to use the constructions resulting from interacting dyads as a trace of the processes that took place in the interaction.
+By looking at the characteristics of the cars built by participants, the authors claim to present "methods for discerning, and quantifying, schema-like intersubjective understandings in material form" [@mcgraw_cultures_2014, p. 4; see also @mitkidis_building_2015; and @wallot_beyond_2016], effectively integrating critiques that I discuss below in @sec:intro-review-criticisms.
+
+The works reviewed here show that there is much room for exploration, on one side, of the methodological choices in transmission chain experiments (some of which will turn out to be more important than others in the trends observed), and on the other, of the theoretical background that sustains a given study.
+Such experiments have therefore much to bring to underlying theoretical debates.
+
+
+#### Language change: Experimental semiotics and Iterated learning
+
+\add{tamariz-experimental-2017}
+
+A related strand of research has developed similar methods to study the emergence and evolution of language.
+A central experimental paradigm is that of iterated learning, which resembles a transmission chain for artificial languages where participants learn a communication system at each step:
+a first group of participants must learn to use a simple artificial language (e.g. an artificial vocabulary for naming a range of objects), after which a second group of participants must learn that language through some interaction with or transmission from the first group.
+The process is then iterated over successive generations, leading to the evolution of the artificial language initially introduced.
+A related setup used in experimental semiotics surfaces the evolution of interaction without transmission across generations.
+It consists in pairing participants and assigning them a task that they must cooperatively solve over repeated iterations, without changing partners.
+Most often, participants can use a communication channel (or must create it) to help in coordination, such that the setup exposes the way participants iteratively develop conventions over the channel.
+Studies using these two paradigms have shown that simple biases in participants' interactions and learning capabilities can lead the final evolved communication system to exhibit non-trivial structure.
+Several factors have been shown to influence the process, including the structure of the objects referred to by the artificial language, the transmission or interaction task (and its surrounding context) for iterated learning, and the reinforcement rules favouring expressivity of the language that will interact with learnability pressures.
+
```

Comment:
TODO
- [ ] abstract out the overarching lines of research in each review section